index.js

```diff
@@ -0,0 +1,13 @@
+const calculateSubTotals = (items) => {
```

Comment:
Remember map calls this function and passes in a single object from the array at a time. So this function will be called multiple times, once for each item. Since it is being handed a single object, `items` is a misleading name. `item` is better since it is one item, not the list of them.